rke/k8s_rke_system_images.go

```diff
@@ -5736,6 +5908,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.18.0",
 		},
+		// Enabled in v2.6.1
```

Comment:
Due to the OOB release and not waiting for v2.6.1, this comment will need to be `Enabled out of band post v2.6.0`